.env

```diff
@@ -3,3 +3,11 @@ MYSQL_USER=contenta
 MYSQL_PASSWORD=contenta
 MYSQL_ALLOW_EMPTY_PASSWORD=yes
 MYSQL_ROOT_PASSWORD=root
+
+HOSTNAME=contenta.local
+HOSTIP=192.168.1.1
+
+HOST_MYSQL_PORT=3336
+HOST_PHP_PORT=9009
+HOST_HTTP_PORT=8888
```

Comment:
Ports maybe good to have by default but why don't use the default ports?

I'm guessing adding a .env.defaults with all this would be better instead and add .env to the gitignore

Reply:
Ya, the defaults would be a good idea to use as a common base, I put those in as I wanted to show an example of slightly modified ports. But I'm fine with whatever.